RIOT-OS/RIOT
# cpu/esp32/stdio_usb_serial_jtag: Fix to ESP32 stdio usb serial hanging if it receives data too quickly

### Contribution description



When the stdio serial usb jtag module of the ESP32 receives data at high speeds, there's a chance its tx fifo won't be flushed and/or the correct irq flag won't be cleared. This leads to nothing being printed through the usb stdio anymore, and (most instances) data being unable to be received, rendering the shell unusable. 

This PR clears the IRQ flags at the end of the ISR and flushes the fifo. With this change I have not been able to repro the bug in question.

### Testing procedure



- Use an ESP32 that has stdio_usb_serial_jtag support. (I've been using a seeedstudio xiao esp32s3)
- Load up a basic example like `examples/default`
- Wail on the shell using the following python code, or anything equivalent:

```
#!/usr/bin/env python3
import serial, time
ser = serial.Serial("/dev/tty.usbmodem111301")
count = 10
while(True):
    ser.write("help\n".encode())
    time.sleep(0.001)
    count -= 1
    if (count == 0):
        break
print("Writer done")
```
- Observe that the shell is no longer responsive
- This PR fixes this issue

### Issues/PRs references
